src/Goerli-DssSpell.t.base.sol

```diff
@@ -287,21 +284,54 @@ contract GoerliDssSpellTestBase is Config, DSTest, DSMath {
             }
             else {
                 // jump to nextCastTime to be a little more forgiving on the spell execution time
-                hevm.warp(prevSpell.nextCastTime());
+                vm.warp(prevSpell.nextCastTime());
                 prevSpell.cast();
             }
         }
     }
+    event Log(string, string);
+    event Log(string, uint256);
 
-    function setUp() public {
-        hevm = Hevm(address(CHEAT_CODE));
+    function stringConcat(string memory a, string memory b) pure internal returns(string memory) {
+        return string(abi.encodePacked(a, b));
+    }
 
+    function readInput(string memory input) internal returns (string memory) {
+        string memory root = vm.projectRoot();
+        string memory path = stringConcat(stringConcat(root, "/config/"), stringConcat(input, ".json"));
+        emit Log("path", path);
```

Comment:
Was this accidentally left in for debug?

Reply:
https://github.com/makerdao/spells-goerli/pull/132/commits/28d3dc4e0d8870cd0b47213b3a1b24014d9f2035